src/app/screen/onboarding/details-form/details-form.component.scss

```diff
@@ -0,0 +1,4 @@
+html, body {
```

Comment:
add this to main styles.css. Here only those css is required which is specific to a particular component. nothing else

Reply:
removed from this component and added in style.css